.github/workflows/ci.yml

```diff
@@ -0,0 +1,116 @@
+name: CI
+on: [push, pull_request]
+
+jobs:
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        jdk: [8]
+        scala: [2.11.12, 2.12.10, 2.13.1]
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v1
+
+      - name: Set up JDK
+        uses: actions/setup-java@v1
+        with:
+          java-version: ${{ matrix.jdk }}
+
+      - name: Compile
+        run: sbt coverage "++${{ matrix.scala }} compile"
+
+      - name: Run tests
+        run: sbt coverage "++${{ matrix.scala }} test"
+
+      - name: Build Scaladoc
+        run: sbt "++${{ matrix.scala }} doc"
+
+      - name: Publish artifact locally
+        run: sbt "++${{ matrix.scala }} publishLocal"
+
+      - name: Compile example project
+        run: cd example && sbt "++${{ matrix.scala }} test"
+
+      - name: Check tut output
+        run: >
+          if [[ "${{ matrix.scala }}" =~ ^2\.12.* ]]; then
```

Comment:
There seems to be an `if` conditional for steps (https://help.github.com/en/actions/automating-your-workflow-with-github-actions/workflow-syntax-for-github-actions#jobsjob_idstepsif) which we should be able to use here.

Reply:
Makes sense, I knew about it but hadn't thought of using it here. Well spotted!